vcsim/README.md

```diff
@@ -78,7 +78,7 @@ flags.  Resources can also be created and removed using the API. In fact, vcsim
 itself uses the vSphere API generate its inventory.
 
 ```console
-$ vcsim -h # pruned to model type flags used in the Examples section
+vcsim -h # pruned to model type flags used in the Examples section
```

Comment:
The nice thing about `console` + `$ ` command prefix is you can clearly see the commands vs output. Compare the current: https://github.com/vmware/govmomi/tree/master/vcsim#usage
The command and comment are colorized black+grey, output is blue. This is esp. useful when a `console` section contains multiple commands+outputs.

To w/o the `$`, all text is blue: https://github.com/vmware/govmomi/blob/c235e4f3282f4f1cee09b7224388b7cd2a30cc9d/vcsim/README.md#usage

I would prefer to keep console+prompt in the sections where include the output.  Ok with me to remove the prompt where we don't include any output. But, in those cases you'd probably want to change `console` to `bash`, so we get the consistent color (black vs blue).

ex see current black: https://github.com/vmware/govmomi/tree/master/vcsim#install-via-go-get
vs pr blue: https://github.com/vmware/govmomi/blob/c235e4f3282f4f1cee09b7224388b7cd2a30cc9d/vcsim/README.md#install-via-go-install

Reply:
Makes sense, wasn't aware of these little subtleties :)